### ISSUE TITLE
Implement statfs operation to report basic stats

### ISSUE THREAD

**Issue by lzw06061139** [awslabs/mountpoint-s3#710]
### Mountpoint for Amazon S3 version

mount-s3 1.3.2（build from main branch）

### Describe the running environment

Running in a ubuntu server（Release：16.04， kernel：4.15.0-142-generic）

### Mountpoint options

```shell
mount-s3  --region us-east-1 --endpoint-url {my-endpoint} --debug --debug-crt -l ~/ {my-bucket} {my-mountpoint}
```

### What happened?

Successfully mounted {my-bucket} on {my-mountpoint}，I can see the mounted filesystem by running commnd:  `cat /proc/mounts` ，but running `df -h`  can't report the mounted filesystem(others can be reported)

**Comment by monthonk** [comment]
Hi @lzw06061139, thanks for reporting the issue. This is expected behavior because Mountpoint does not report file system stats to the kernel, so `df -h` would filter it out. However, you should still be able to see the mount by running `df -a` command.

I willl change this to feature request to implement `statfs` operation (https://docs.rs/fuser/latest/fuser/trait.Filesystem.html#method.statfs) which should make `df -h` includes Mountpoint by default.

**Comment by lzw06061139** [comment]
> Hi @lzw06061139, thanks for reporting the issue. This is expected behavior because Mountpoint does not report file system stats to the kernel, so `df -h` would filter it out. However, you should still be able to see the mount by running `df -a` command.
> 
> I willl change this to feature request to implement `statfs` operation to report basic stats (https://docs.rs/fuser/latest/fuser/trait.Filesystem.html#method.statfs) so `df -h` would include Mountpoint in its output by default.

 Thank you for your reply @monthonk . `df -a` can see the mount with zero disk space usage. By the way,when do you expect the new feature `statfs` will be implemented?

**Comment by monthonk** [comment]
Sorry, we don't have any target date yet. But we will let you know in this thread once we have something to share.

**Comment by dannycjones** [comment]
This was implemented in #1118 and released in [Mountpoint v1.12.0](https://github.com/awslabs/mountpoint-s3/releases/tag/mountpoint-s3-1.12.0)! 🚀
